core/server/models/post.js

```diff
@@ -5,10 +5,7 @@ var _              = require('lodash'),
     sequence       = require('../utils/sequence'),
     errors         = require('../errors'),
     Showdown       = require('showdown-ghost'),
-    ghostgfm       = require('../../shared/lib/showdown/extensions/ghostgfm'),
-    footnotes      = require('../../shared/lib/showdown/extensions/ghostfootnotes'),
-    highlight      = require('../../shared/lib/showdown/extensions/ghosthighlight'),
-    converter      = new Showdown.converter({extensions: [ghostgfm, footnotes, highlight]}),
+    converter      = new Showdown.converter({extensions: ['ghostgfm', 'footnotes', 'highlight']}),
```

Comment:
All 4 extensions are now part of showdown proper, so yep :) 
Ref: https://github.com/showdownjs/showdown#extensions